Sources/MapboxCoreNavigation/NativeHandlersFactory.swift

```diff
@@ -53,7 +53,8 @@ class NativeHandlersFactory {
     
     lazy var navigator: MapboxNavigationNative.Navigator = {
         onMainQueueSync { // Make sure that Navigator pick ups Main Thread RunLoop.
-            LogConfiguration.setLoggingLevelForCategory("*", level: LoggingLevel.info)
+            let loggingLevel = NSNumber(value: LoggingLevel.info.rawValue)
+            LogConfiguration.setLoggingLevelForCategory("*", upTo: loggingLevel)
```

Comment:
Is this API really working? I've changed to other logging levels and do not see any difference (e.g. still getting debug related logs from Navigation Native regardless of provided `loggingLevel` value).

Reply:
It definitely used to. Debug messages were quite spammy, logging basically any NN method enter/leave. @mapbox/navnative did you update something here? v106 changelog does not mention logging changes.

Reply:
These are the changes on the core-sdk side - the logging API in v22.0.0 is different.

Reply:
I see, but Maxim said that debug messages are still appearing even though log level is set higher. Do you see same behavior on your side? Should we ping core team?